### ISSUE TITLE
Sticky Tabs break Back to top button

### ISSUE THREAD

**Issue by jacobswe** [squidfunk/mkdocs-material#2807]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [x] ... there are no open or closed issues that are related to my problem

### Description

When using sticky tabs and back to top button, clicking the back to top button does not disappear after scrolling finished.

### Expected behaviour

Back to top button should disappear after clicking and scrolling.

![image](https://user-images.githubusercontent.com/14115132/124398193-6f1fb280-dce2-11eb-9fe3-1874d6347d29.png)

### Actual behaviour

Back to top button does not disappear and stays blue.

![image](https://user-images.githubusercontent.com/14115132/124398016-77c3b900-dce1-11eb-879d-2a85a16d9799.png)

### Steps to reproduce

1. Use sticky tabs and top theme features
2. Click back to top button

### Package versions

- Python: 3.7.10
- MkDocs: 1.1.2
- Material: 7.1.9+insiders.2.9.2

### Configuration

```yaml
site_name: My Docs
theme:
  features:
  - navigation.tabs
  - navigation.tabs.sticky
  - navigation.top
  name: material
```

### System information

- Operating system: Ubuntu 18.04.5
- Browser: Chrome Version 91.0.4472.124 (Official Build)

**Comment by squidfunk** [comment]
Thanks for reporting. I can confirm that this happens when using sticky tabs without an announcement bar or anything else that ensures the header is always in the same position. This is now fixed in the latest commit on the Insiders repository. I'll issue a new release this weekend!

**Comment by squidfunk** [comment]
The fix was just released as part of 7.1.10 (+ Insiders)
